instance/github.py

```diff
@@ -86,6 +87,19 @@ def get_settings_from_pr_body(pr_body):
         return ''
 
 
+def is_pr_body_requesting_ephemeral_databases(pr_body, domain):
+    """
+    Return True if the PR body specified that the sandbox should use ephemeral
+    databases, False if it specifies persistent databases, or None otherwise
+    """
+    escaped_domain = re.escape(domain)
+    if re.search(r'{0}.*\(ephemeral databases?\)'.format(escaped_domain), pr_body):
+        return True
+    if re.search(r'{0}.*\(persistent databases?\)'.format(escaped_domain), pr_body):
```

Comment:
Can you remove the parenthesis in the regex? This will allow to have several options on the same line, like `... (ephemeral databases, other option)`